### PR TITLE
added variables and locals for vpc

### PR DESCRIPTION
**Pull request by kunskytested**
problem
needed variables and locals

solution
added variables and locals

### CODE REVIEW

File: locals.tf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+locals {
+  vpc_id = aws_vpc.cloudpros_vpc.id
+  create_vpc = var.create_vpcs
```
Comment (tberry147) [inline locals.tf]: Why do we need this here?

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+variable "cidr_block" {
+  description = "The CIDR of the VPC"
+  type = string
+  default = "10.0.0.0/16"
+}
+variable "create_vpc" {
```
Comment (tberry147) [inline variables.tf]: Same here too?

**Review by tberry147 — CHANGES_REQUESTED** [review]
Please explain those codes you have there

File: locals.tf (hunk 1 above)
Comment (ghost) [inline locals.tf]: Take off this line

File: variables.tf (hunk 2 above)
Comment (ghost) [reply variables.tf]: Same here

**Review by ghost — CHANGES_REQUESTED** [review]
see comment

**Review by Mummycutie — APPROVED** [review]
LGTM

**Review by tberry147 — APPROVED** [review]
LGTM

**Review by ghost — APPROVED** [review]
Looks perfect. Good job

**Pull request merged**